R/summary_functions.R

```diff
@@ -162,17 +162,22 @@ print.coverage <- function(x, group = c("filename", "functions"), by = "line", .
 
   overall_percentage <- percent_coverage(df, by = by)
 
-  message(crayon::bold(
-      paste(collapse = " ",
-        c(attr(x, "package")$package, to_title(type), "Coverage: "))),
-    format_percentage(overall_percentage))
-
+  cli::cli_inform(paste0(
+    cli::style_bold(
+      "{attr(x, 'package')$package} {to_title(type)} Coverage: "
+    ),
+    format_percentage(overall_percentage)
+  ))
   by_coverage <- percents[order(percents,
       names(percents))]
 
```

Comment:
This was the original intent of my PR. Import cli instead of crayon. See the visual change in the PR main comment